cadence/worker.py

```diff
@@ -15,7 +15,15 @@
 
 @dataclass
 class WorkerOptions:
-    pass
+    max_concurrent_activity_execution_size: int = 1000
+    worker_activities_per_second: float = 100000
+    max_concurrent_local_activity_execution_size: int = 1000
+    worker_local_activities_per_second: float = 100000
+    task_list_activities_per_second: float = 100000
+    max_concurrent_activity_task_pollers: int = 2
+    max_concurrent_decision_task_execution_size: int = 1000
+    worker_decision_tasks_per_second: float = 100000
+    max_concurrent_decision_task_pollers: int = 2
```

Comment:
If you pass null values, does it default to the server defaults? because I don’t want to codify the defaults here if the server-side will potentially change them in the future.